api/main.py

```diff
@@ -70,11 +70,14 @@ def update_user_data(user_id):
     house_name = data['house_name']
     house_location = data['location']
 
-    if any in data is not None:
-        data.update({ name:'name', password: 'password', house_name: 'house_name',
-                      house_location: {'lat': 51.529249, 'lng': -0.117973, 'description': 'University of Bristol'}})
+    if data is not None:
+     api.user_repository.update_user_account = data.update({ name:'name'
```

Comment:
Why do you use hardcoded 'password' etc? Why don't you use the values from 'data'?